package/MDAnalysis/analysis/rms.py

```diff
@@ -527,7 +527,7 @@ def save(self, filename=None):
         if filename is not None:
             if self.rmsd is None:
                 raise NoDataError("rmsd has not been calculated yet")
-            np.savetxt(filename, self.rmsd)
+            np.save(filename, self.rmsd)
```

Comment:
Why did you change this? This can break code for users, don't do this light-heartedly! Please revert!

save and savetxt are two very different things.